Makefile.am

```diff
@@ -327,6 +327,15 @@ yamllint:
 	done; \
 	echo "-----------"
 
+# Build & lint documentation.
+#
+.PHONY: doclint
+doclint:
+	echo -e "\nBuild and lint documentation"; \
+	echo "-----------"; \
+	$(MAKE) -C $(srcdir)/doc/ html SPHINXOPTS="-W"; \
```

Comment:
This won't report any errors if the user has built documentation before. You have to clean first, then build.

I'm not sure it's a good idea to have this part of fastcheck. As soon as we have more documentation, it can easily take a minute or two to rebuild the documentation.

Reply:
OK, I've added the clean step and removed it from fastcheck.